examples/api-gateway/README.md

```diff
@@ -0,0 +1,79 @@
+# Example using the AWS C++ Lambda runtime and Amazon API Gateway
+
+In this example, we'll build a simple "Hello, World" lambda function that can be invoked using an api endpoint created using Amazon API gateway. This example can be viewed as the C++ counterpart to the NodeJS "Hello, World" API example as viewed [here](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-lambda.html). At the end of this example, you should be able to invoke your lambda via an api endpoint and receive a raw JSON response. This example employs the use of the AWS C++ SDK to parse the request and write the necessary response. 
```

Comment:
Using the C++ SDK just to do JSON handling seems excessive. However, all other solutions will have similar or worse downsides. I think because this is an example, and in practice, it's more likely that users will be using the AWS C++ SDK anyway, I think this is OK.

Reply:
I thought the same thing initially! I didn't intend to use the AWS C++ SDK but considered the use case and that the end user would be using the SDK.